client-app/src/examples/contact/mobile/details/DetailsPanelModel.ts

```diff
@@ -21,6 +18,11 @@ export default class DetailsPanelModel extends HoistModel {
         return !this.formModel.readonly;
     }
 
+    get currentContact() {
+        if (XH.routerState?.name !== 'default.details') return null;
+        return this.appModel.contacts.find(it => it.id === XH.routerState.params.id);
```

Comment:
Rather than giving this its own observable state, we can pull the currentContact straight out of AppModel.contacts. This helps ensure the contact info is always synchronised throughout the app.